vllm/engine/llm_engine.py

```diff
@@ -974,3 +990,45 @@ def list_loras(self) -> List[int]:
 
     def check_health(self) -> None:
         self.model_executor.check_health()
+
+    def create_trace_span(self, seq_group: SequenceGroup, now: float) -> None:
+        if seq_group.trace_context is None:
+            return
+        arrival_time_nano_seconds = int(seq_group.metrics.arrival_time * 1e9)
+        with tracer.start_as_current_span(
+                "llm_request",
+                kind=SpanKind.SERVER,
+                context=seq_group.trace_context,
+                start_time=arrival_time_nano_seconds) as seq_span:
+            metrics = seq_group.metrics
+            ttft = metrics.first_token_time - metrics.arrival_time
+            e2e_time = now - seq_group.metrics.arrival_time
+            # attribute names are based on
+            # https://github.com/open-telemetry/semantic-conventions/blob/main/docs/gen-ai/llm-spans.md
+            seq_span.set_attribute("gen_ai.response.model",
```

Comment:
Can I suggest using the constants from https://github.com/traceloop/openllmetry/blob/main/packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv/ai/__init__.py
?
We keep them up to date with the decisions we're making in the otel semantic conventions working group

Reply:
Thanks for your suggestion! I've added a dependency on `opentelemetry-semantic-conventions-ai` and extended `SpanAttributes` with the missing constants.